management_api_app/core/config.py

```diff
@@ -30,3 +30,12 @@
 
 # Logging and monitoring
 APP_INSIGHTS_INSTRUMENTATION_KEY: str = config("APPINSIGHTS_INSTRUMENTATIONKEY", default="")
+
+# Authentication
+API_CLIENT_ID: str = config("API_CLIENT_ID", default="")
+API_CLIENT_SECRET: str = config("API_CLIENT_SECRET", default="")
+SWAGGER_UI_CLIENT_ID: str = config("SWAGGER_UI_CLIENT_ID", default="")
+AAD_TENANT_ID: str = config("AAD_TENANT_ID", default="")
+
+AAD_INSTANCE: str = config("AAD_INSTANCE", default="https://login.microsoftonline.com")
+API_AUDIENCE: str = config("API_AUDIENCE", default=f"api://{API_CLIENT_ID}")
```

Comment:
Might be a good idea to document these.

Reply:
Indeed - good catch!

Reply:
Added #357 to document these and all other ENV VARs needed for local runs of API - will pick that up next